src/main/java/org/titaniumtitans/frc2022/subsystems/Climber.java

```diff
@@ -35,8 +34,8 @@ public Climber() {
     m_leftClimber = new TalonFX(ClimberConstants.kLeftCimberPort);
     m_rightClimber = new TalonFX(ClimberConstants.kRightClimberPort);
 
-    m_rightMaxHeight = PropertyManager.createDoubleProperty(false, "Right Max Height", 0.0);
-    m_leftMaxHeight = PropertyManager.createDoubleProperty(false, "Left Max Height", 0.0);
+    m_rightMaxHeight = new GosDoubleProperty(false, "Right Max Height", 0.0);
+    m_leftMaxHeight = new GosDoubleProperty(false, "Left Max Height", 0.0);
 
     m_rightClimber.setInverted(InvertType.InvertMotorOutput);
 
```

Comment:
Tip for the line below this, which isn't shown I would say with 99% certainty that your left and right will have the same PID constants. Its a little odd, but you can make it so there is one entry in the preferences / network table, but it operates on both motor controllers if you use the same name. We do this in our [chassis](https://github.com/GirlsOfSteelRobotics/GirlsOfSteelFRC/blob/main/y2022/RapidReact/src/main/java/com/gos/rapidreact/subsystems/ChassisSubsystem.java#L228)